### PR TITLE
Introduce profile term into TD model

### PR DESCRIPTION
**Pull request by sebastiankb**
Also see #806 and https://github.com/w3c/wot-profile/pull/79

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1121.html" title="Last updated on May 3, 2021, 9:46 PM UTC (49fd9ef)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1121/0e54c8f...49fd9ef.html" title="Last updated on May 3, 2021, 9:46 PM UTC (49fd9ef)">Diff</a>

### CODE REVIEW

File: validation/td-validation.ttl [hunk 1]
```diff
@@ -93,7 +93,7 @@
                 is applied to TD instances."""^^rdf:HTML ;
         sh:datatype xsd:anyURI ;
         sh:maxCount 1 ;
-        sh:order 8 ;
+        sh:order 9 ;
```
Comment (danielpeintner) [inline validation/td-validation.ttl:96]: Note: This change is unrelated but I noticed afterwards follows yet another `sh:property` without `sh:order` and the one after that one starts with `sh:order 10` ;

It seems to be that we might need to add `sh:order` always and shift the numbers of the others? Am I wrong?

**Comment by sebastiankb** [comment]
from today's TD call we decided to merge

File: index.html [hunk 2]
```diff
@@ -1128,7 +1128,8 @@ <h2>Core Vocabulary Definitions</h2>
 <tr class="rfc2119-table-assertion" id="td-vocab-securityDefinitions--Thing"><td><code>securityDefinitions</code></td><td>Set of named security configurations
                 (definitions only). Not actually applied unless
                 names are used in a <code>security</code>
-                name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr></tbody></table><p><span class="rfc2119-assertion" id=
+                name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-profile--Thing"><td><code>profile</code></td><td>Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation.</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a> or <a>Array</a> of <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a></td></tr></tbody></table><p><span class="rfc2119-assertion" id=
```
Comment (benfrancis) [inline index.html:1132]: Nit: The profile member doesn't indicate a profile mechanism followed by a Thing Description, it _is_ a mechanism inside a Thing Description to indicate that the Web Thing it describes conforms to a particular profile or profiles.

Therefore I probably would have worded this slightly differently, e.g. "Indicates a profile or list of profiles to which the Web Thing conforms". Ideally with a link to the WoT Profile specification, if we can do that at this point.

**Review by benfrancis — COMMENTED** [review]
Sorry for the late review.

I agree with this change, I just might have worded the description slightly differently.

**Pull request merged**